docs/build/js-client/2-installation.md

```diff
@@ -18,23 +18,25 @@ Adding the Fluence JS client for your web application is very easy.
    </head>
    ```
 
-   If you cannot or don't want to use a CDN, feel free to get the script directly from the [npm package](https://www.npmjs.com/package/@fluencelabs/js-client.web.standalone) and host it yourself. You can find the script in the `/dist` directory of the package. (Note: this option means that developers understand what they are doing and know how to serve this file from their own web server.)
+   If you cannot, or don't want to, use a CDN, feel free to get the script directly from the [npm package](https://www.npmjs.com/package/@fluencelabs/js-client.web.standalone) and host it yourself. You can find the script in the `/dist` directory of the package.
```

Comment:
I am afraid the first two commas should be omitted.

Reply:
non-restrictive clause warrants comma(s)